Lib/test/test_signal.py

```diff
@@ -1348,6 +1348,8 @@ def handler(signum, frame):
         # Python handler
         self.assertEqual(len(sigs), N, "Some signals were lost")
 
+
+    @support.requires_resource('flaky')
```

Comment:
Maybe I messed up my suggestion, but let's just revert `test_signal.py` in its all. There is no fully flaky test in this one (flakiness is only on free-threaded build and I don't know whether we have something to mark something flaky on a special build only)